README.md

```diff
@@ -49,36 +63,166 @@ const { MediaContextProvider, Media, createMediaStyle } = createMedia({
+React attaches, it simply washes over and removes markup that is unneeded, via a
+`matchMedia` call.
+
+This is a unique solution to the long-standing problem of how best to design
+for, and respond to, layouts that can be viewed on any kind of device, and at
+any size, but which also need to be server-side rendered.
+
+### SSR Example
+
+First, configure `@artsy/fresnel` in a `Media` file that can be shared across
+the app:
+
+```tsx
+// Media.tsx
+
+import { createMedia } from "@artsy/fresnel"
+
+const ExampleAppMedia = createMedia({
+  breakpoints: {
+    xs: 0,
+    sm: 768,
+    md: 1000,
+    lg: 1200,
+  },
+})
+
+// Generate CSS to be injected into the head
+export const mediaStyle = ExampleAppMedia.createMediaStyle()
+export const { Media, MediaContextProvider } = ExampleAppMedia
+```
+
+Create a new `App` file which will be the launching point for our application:
+
+```tsx
+// App.tsx
+
+import React from "react"
+import { Media, MediaContextProvider } from "./Media"
+
+export const App = () => {
+  return (
     <MediaContextProvider>
-      <Media at='sm'>
-        <MobileApp />
-      </Media>
-      <Media at='md'>
-        <TabletApp />
-      </Media>
-      <Media greaterThanOrEqual='lg'>
-        <DesktopApp />
-      </Media>
+      <Media at="xs">Hello mobile!</Media>
+      <Media greaterThan="xs">Hello desktop!</Media>
     </MediaContextProvider>
-  </>
-)
+  )
+}
 ```
 
-### Demo
+Mount `<App />` on the client:
+
+```tsx
+// client.tsx
 
-You can find an example in this repository and run it locally like so:
+import React from "react"
+import ReactDOM from "react-dom"
+import { App } from "./App"
 
+ReactDOM.render(<App />, document.getElementById("react"))
 ```
-git clone https://github.com/artsy/react-responsive-media.git
-cd react-responsive-media
-yarn install
-yarn example
-open http://localhost:8080
+
+Then on the server, setup SSR rendering and pass `mediaStyle` into a `<style>`
+tag in the header:
+
+```tsx
+// server.tsx
+
+import React from "react"
+import ReactDOMServer from "react-dom/server"
+import express from "express"
+
+import { App } from "./App"
+import { mediaStyle } from "./Media"
+
+const app = express()
+
+app.get("/", (_req, res) => {
+  const html = ReactDOMServer.renderToString(<App />)
+
+  res.send(`
+    <html>
+      <head>
+        <title>@artsy/fresnel - SSR Example</title>
+
+        <!–– Inject the generated styles into the page head -->
+        <style type="text/css">${mediaStyle}</style>
+      </head>
+      <body>
+        <div id="react">${html}</div>
+
+        <script src='/assets/app.js'></script>
+      </body>
+    </html>
+  `)
+})
+
+app.listen(3000, () => {
+  console.warn("\nApp started at http://localhost:3000 \n")
+})
 ```
 
-### Why not conditionally render?
+And that's it! To test, disable JS and scale your browser window down to a
+mobile size; it will correctly render without the need to use a user-agent or
```

Comment:
mobile size;  `the page` will correctly render without? Again takes a second to realize what `it ` refers to:)